### ISSUE TITLE
Character and Corporation Wallet Journal filter blank

### ISSUE THREAD

**Issue by ditzyscot** [eveseat/seat#145]
Problem: When trying individual character wallet journals or corporation wallet journals, the transaction filter drop down is blank, the column in the table regarding the wallet journal is also blank

Expected: Expected a list filter options to appear in drop down based on overall pulled wallet history? And the wallet journal column to be populated with something

I have a few screenshots of the issue below, showing the blank transaction filter box, what occurs when you select a blank list option, and finally what occurs when you click filter.

[http://i.imgur.com/LlPMUEh.png](http://i.imgur.com/LlPMUEh.png)

[http://i.imgur.com/iao7Q6a.png](http://i.imgur.com/iao7Q6a.png)

[http://i.imgur.com/pVCC3D7.png](http://i.imgur.com/pVCC3D7.png)

I've pasted the version info from artisan admin diagnose command:

```
+---------------+---------------+---------------+
| Package Name  | Local Version | Latest Github |
+---------------+---------------+---------------+
| Api           | 1.0.13        | 1.0.13        |
| Console       | 1.0.17        | 1.0.17        |
| Eveapi        | 1.0.18        | 1.0.18        |
| Notifications | 1.0.4         | 1.0.4         |
| Web           | 1.0.31        | 1.0.31        |
| Services      | 1.0.23        | 1.0.23        |
+---------------+---------------+---------------+
```

I've had a search through closed issues and nothing resembling this came up, so hope this is an easy one to resolve, happy to report everything else functions as expected.

**Comment by warlof** [comment]
Hello,

What is your sde version ? It's displayed bottom right corner on page or in settings.

Be sûre you've run `php artisan eve:update-sde`

**Comment by ditzyscot** [comment]
Hi,

The sde version is: SDE Version sde-20161011-TRANQUILITY

I've given that command a run as well since you mentioned it, and the output confirmed it was the latest SDE version.

Cheers

**Comment by leonjza** [comment]
This bug was fixed with https://github.com/eveseat/eveapi/commit/bcd15e024239b465a1ef666234acc55d37085f7b by @johnnysplunk and should land in the next version of SeAT.

**Comment by leonjza** [comment]
Fixed as part of the 2.0.0 release.
